Pyfhel/PyPtxt.pyx

```diff
@@ -248,6 +249,8 @@ cdef class PyPtxt:
         """returns the integer in the first slot of the plaintext"""
         if (self._scheme != scheme_t.bfv):

```

Comment:
scheme_t.bgv would raise this exception. change to have both conditions in one single `if` clause (`if a and b`)